### DISCUSSION TITLE
NUMLOCK doesn't work

### DISCUSSION THREAD

**Discussion by tunaflsh** [mpvnet-player/mpv.net#491] [Q&A] [answered]
I don't know if this is a feature or not. It doesn't distinguish numpad keys with numlock ON or OFF.
When testing inputs mpvnet always sees numpad keys as KP* regardless of numlock state.
When configuring with input editor with numlock ON it recognizes them as KP*, with numlock OFF it sees LEFT, RIGHT, UP, DOWN, HOME, END, etc.

**Comment by stax76** [comment]
In the bindings editor, the code is from me, in the main window the input information is passed directly to the mpv API without any interpretation or modification, so it is likely you get the same behavior in mpv.

Do you sometimes or often change the numlock state? I would guess that most people that have a numpad have it always enabled.

**Reply by tunaflsh** [reply]
Thank you for answering. I change numlock state because my arrows are broken. But I can live with setting the numpad keys to bind with the commands, it's not like I bind numbers to anything.

**Comment by tunaflsh** [comment, marked as answer]
I ended up binding all numpad keys to the same commands as arrows. Then at some point I used autohotkey to map numpad keys into normal arrows. Solving this problem altogether.
